kolibri/core/auth/api.py

```diff
@@ -85,9 +85,13 @@ class KolibriAuthPermissionsFilter(filters.BaseFilterBackend):
     to the ``filter_readable`` method on ``KolibriAbstractBaseUser``.
     """
 
+    def __init__(self, endpoints=None):
+        self.endpoints = endpoints or ["-list"]
```

Comment:
I am sure there is a clever DRF way to detect whether something is a list endpoint or not - but it's not coming to mind...

Reply:
Found it